ffi/build.py

```diff
@@ -134,8 +134,8 @@ def main_posix(kind, library_ext):
         print(warning + '\n')
     else:
 
-        if not out.startswith('9.0.'):
-            msg = ("Building llvmlite requires LLVM 9.0.x, got "
+        if not out.startswith('10.0.'):
```

Comment:
Are we going to allow LLVM9 for Aarch64?

Reply:
yes we are, and I have just made the change in 324533e56d8e2206484b5daf10565f64ade0869e